consensus/hotstuff/model/timeout_bench_test.go

```diff
@@ -0,0 +1,82 @@
+package model_test
+
+import (
+	"bytes"
+	"testing"
+
+	clone "github.com/huandu/go-clone/generic" //nolint:goimports
```

Comment:
why is the linter directive needed?

Reply:
👍 removed in https://github.com/onflow/flow-go/pull/7314/commits/a1e68d972ed77145d63541db45e490c31b57b98e